python/Completion.py

```diff
@@ -8,18 +8,31 @@ def get_completions(self, column, partialWord):
         parameters['WantDocumentationForEveryCompletionResult'] = \
             bool(int(vim.eval('g:omnicomplete_fetch_full_documentation')))
 
+        want_snippet = \
+            bool(int(vim.eval('g:omnicomplete_want_snippet')))
+
+        want_method_header = \
+            bool(int(vim.eval('g:omnicomplete_want_method_header')))
+
+        want_return_type = \
+            bool(int(vim.eval('g:omnicomplete_want_return_type')))
```

Comment:
Ok, I will do that. Thanks. 

-----Original Message-----
From: "thinca" <notifications@github.com>
Sent: ‎04/‎05/‎2015 16:25
To: "OmniSharp/omnisharp-vim" <omnisharp-vim@noreply.github.com>
Cc: "Mark Woodhall" <mark.woodhall@gmail.com>
Subject: Re: [omnisharp-vim] #206 Support for on the fly generation andexpansion of snippets during completion (#209)

In python/Completion.py:
> @@ -8,18 +8,31 @@ def get_completions(self, column, partialWord):
>          parameters['WantDocumentationForEveryCompletionResult'] = \
>              bool(int(vim.eval('g:omnicomplete_fetch_full_documentation')))
>  
> +        want_snippet = \
> +            bool(int(vim.eval('g:omnicomplete_want_snippet')))
> +
> +        want_method_header = \
> +            bool(int(vim.eval('g:omnicomplete_want_method_header')))
> +
> +        want_return_type = \
> +            bool(int(vim.eval('g:omnicomplete_want_return_type')))
Please use g:OmniSharp_ prefix for config variable.
g:omnicomplete_fetch_full_documentation exists already, but this is wrong. I'll fix it.
—
Reply to this email directly or view it on GitHub.